libs/utils/src/hooks/useFetchOmdb.tsx

```diff
@@ -0,0 +1,30 @@
+import { Film } from '@form-exercise/data/interface';
+import axios, { AxiosError } from 'axios';
+import { useQuery } from 'react-query';
+
+interface UseFetchOmdb {
+  searchBy: string;
+  type?: string;
+}
+
+interface Films {
+  Reponse: string;
+  Search: Film[];
+  totalResults: string;
+}
+
+export function useFetchOmdb({ searchBy, type = 'movie' }: UseFetchOmdb) {
+  const getFilms = async (): Promise<Films> => {
+    const { data } = await axios(
+      `http://www.omdbapi.com/?apikey=${process.env['NX_OMDB_API_KEY']}&s=${searchBy}&type=${type}`
+    );
+    return data;
+  };
+
+  const { isLoading, data, error } = useQuery<Films, AxiosError>(
+    'films',
+    getFilms
+  );
+
+  return { isLoading, data, error };
```

Comment:
you could return all the rest of variables from useQuery, it will save some time when you need to access a specific function/variable that is not presently returned
```tsx
  const query = useQuery<Films, AxiosError>(
    'films',
    getFilms
  );

  return query;
```